src/extra/misc/PointcloudRasteriser.cpp

```diff
@@ -213,6 +215,28 @@ namespace roofer::misc {
     }
   }
 
+  bool assessGlassRoof(const ImageMap& pc) {
+    auto& grp = pc.at("grp").array;
```

Comment:
this method takes into account only the mean value of all cells. Anything below 0.75  is considered a glass roof.